e2e/pods_test.go

```diff
@@ -7,7 +7,10 @@ import (
 )
 
 func TestPodLifecycle(t *testing.T) {
-	spec, err := fixtures.ReadFile("fixtures/nginx.yml")
+	// delete the pod first
+	kubectl("delete", "pod/vk-e2e-hpa")
+
+	spec, err := fixtures.ReadFile("fixtures/hpa.yml")
```

Comment:
If we aren't going to use the other fixture, can you delete it?

Reply:
deleted unused fixture.